src/contexts/App.tsx

```diff
@@ -5,8 +5,7 @@ export interface AppProviderProps {
 }
 
 const initialState = {
-  container: HTMLDivElement,
-  mounted: false
+  container: null as HTMLDivElement
```

Comment:
by initializing this to a falsey value, we can reliably use `container` to determine if the map component mounted.

Reply:
that's what i meant, thanks

Reply:
we don't need to set the `view.container` to `null` on a cleanup because if the DOM node is removed it's automatically null. We could explicitly do it, but it's not needed.